config/localizations_native/en/app.json

```diff
@@ -63,6 +63,7 @@
       "NAME": "Name",
       "NETWORK": "Network",
       "NEWS": "News",
+      "NEW": "New",
```

Comment:
There is already a "NEW" in config/localizations/en/app.json (APP.GENERAL.NEW)

Reply:
good catch, unsure how i missed that 🤔 
addressed.